wp4-compat-js/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "fa-wp4-compat",
+  "version": "4.0.2-rc1",
+  "private": true,
+  "dependencies": {
+    "@wordpress/api-fetch": "^5.1.1",
+    "@wordpress/components": "14.1.10",
+    "@wordpress/compose": "4.1.5",
+    "@wordpress/data": "5.1.5",
+    "@wordpress/element": "3.1.1",
+    "@wordpress/i18n": "^3.16.0",
+    "@wordpress/icons": "4.0.2",
+    "@wordpress/primitives": "2.1.1",
+    "@wordpress/rich-text": "4.1.5",
+    "react": "16.13.1",
+    "react-dom": "16.13.1",
+    "react-native": "0.63.4"
+  },
+  "devDependencies": {
+    "@wordpress/babel-preset-default": "6.2.0",
+    "@wordpress/scripts": "16.1.4",
+    "babel-plugin-lodash": "^3.3.4"
+  },
+  "scripts": {
+    "build": "wp-scripts build --webpack-no-externals"
```

Comment:
Ok. So this is the trick to force the v4 compat build to include React and whatnot?

Reply:
Wait...is `--webpack-no-externals` needed?

Reply:
According to the [npm docs](https://www.npmjs.com/package/@wordpress/scripts):

> --webpack-no-externals – disables scripts' assets generation, and omits the list of default externals.

The reason I'm using it is to omit the list of default externals. `wp-scripts` externalizes `react` and all of the `@wordpress` packages by default. We want to opt out of those defaults for this compat.js bundle because this is precisely the place where we _don't_ want those modules to be externalized.

And I also used `--webpack-no-externals` in the `admin/package.json` because while we do want to externalize these modules in that bundle, I want to customize _how_ they are externalized: namely, using our own global.